synapse/config/consent_config.py

```diff
@@ -42,6 +42,14 @@
 # until the user consents to the privacy policy. The value of the setting is
 # used as the text of the error.
 #
+# 'require_at_registration', if enabled, will add a step to the registration
+# process, similar to how captcha works. Users will be required to accept the
+# policy before their account is created.
+#
+# 'policy_name' is the name of the policy users will see when registering for
```

Comment:
I really didn't understand what "the name of the policy users will see" meant: is it a filename? Turns out it's not, but please could you be clearer.